packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -149,19 +149,19 @@ const defaultProps = {
   timeInputAttributes: undefined,
   value: undefined,
   timeVariant: DateTimeUtils.FORMAT_24_HOUR,
-  timeZone: DateTimeUtils.getLocalTimeZone(),
+  initialTimeZone: DateTimeUtils.getLocalTimeZone(),
 };
 
 class DateTimePicker extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      dateTime: DateTimeUtils.createSafeDate(props.value, props.timeZone),
+      dateTime: DateTimeUtils.createSafeDate(props.value, props.initialTimeZone),
```

Comment:
If we don't want this to be able to change, should this be stored on the DateTimePicker and the inital value always be referenced?
```
class DateTimePicker extends React.Component {
  constructor(props) {
    super(props);
    this.timezone = props.initialTimeZone

```

Reply:
Changed here ea2630e205a708f19fe9d9c16c9f9cbb247d0434